### DISCUSSION TITLE
Goals and features for release `3.3.0`

### DISCUSSION THREAD

**Discussion by TimothyMakkison** [riok/mapperly#990] [General]
It's been about 3 months since the release of `3.2.0`, since then we've added several new bug fixes, qol changes and major features such as private property mapping. Are there any particular issues that should be prioritised? I see that there aren't any milestones for it.

And congrats on the 2k stars 😄

**Comment by latonz** [comment]
Time flies 🙈 We just released 3.3.0 😊 Thanks for all your contributions!
